src/PortingAssistant.Client.Client/FileParser/GitConfigFileParser.cs

```diff
@@ -32,7 +34,16 @@ public static string getGitRepositoryUrl(string gitRepositoryRootPath)
             }
 
             var repo = new Repository(gitRepositoryRootPath);
```

Comment:
I meant to put a try/catch starting from this line.. Repository class initialization throws many exception...